chat-widget/src/components/livechatwidget/common/liveChatConfigUtils.ts

```diff
@@ -1,7 +1,16 @@
+import { ConversationMode } from "../../../common/Constants";
+
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export const isPostChatSurveyEnabled = async (chatSDK: any) : Promise<boolean> => {
     const chatConfig = await chatSDK.getLiveChatConfig();
     const postChatEnabled = chatConfig.LiveWSAndLiveChatEngJoin
         ?.msdyn_postconversationsurveyenable.toString().toLowerCase();
     return postChatEnabled === "true";
-};
\ No newline at end of file
+};
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const isPersistentChatEnabled = async (chatSDK: any): Promise<boolean> => {
+    const chatConfig = await chatSDK.getLiveChatConfig();
+    const conversationMode = chatConfig.LiveWSAndLiveChatEngJoin?.msdyn_conversationmode.toString().toLowerCase();
```

Comment:
do we really need to pass the whole chatsdk to validate 1 attribute of the config?, it seems excessive